drivers/mipi_dbi/mipi_dbi_spi.c

```diff
@@ -65,8 +65,7 @@ static int mipi_dbi_spi_write_helper(const struct device *dev,
 		return ret;
 	}
 
-	if (dbi_config->mode == MIPI_DBI_MODE_SPI_3WIRE &&
-	    IS_ENABLED(CONFIG_MIPI_DBI_SPI_3WIRE)) {
+	if (dbi_config->mode == MIPI_DBI_MODE_SPI_3WIRE) {
```

Comment:
`CONFIG_MIPI_DBI_SPI_3WIRE` was probably added to reduce the memory footprint of the driver if 3-wire mode isn't needed. If you're going to remove the check here, then you need to remove the symbol definition from `drivers/mipi_dbi/Kconfig.spi` too.

Reply:
Correct, reverted, thanks.